<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Review</class>
 <widget class="QDialog" name="Review">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Review{
border-image:url(:/Image/登录界面.png);
}
</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-1</y>
     <width>721</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#tab_waiter
{
	border-image:url(:/Image/登录界面2.png);
	border-radius:0;
}
QWidget#tab_dishes
{
	border-image:url(:/Image/登录界面2.png);
	border-radius:0;
}
</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_waiter">
    <attribute name="title">
     <string>评价服务员</string>
    </attribute>
    <widget class="QPushButton" name="star1_pbt">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>260</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="star2_pbt">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>260</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="star3_pbt">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>260</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="star5_pbt">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>260</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="star4_pbt">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="waiter_textEdit">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>270</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(244,244,244,100);
font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>145</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>服务员ID：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>145</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QListView" name="listView_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>211</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(254, 254, 254, 50);
border-radius:0;</string>
     </property>
    </widget>
    <widget class="QListView" name="listView_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>255</y>
       <width>295</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(254, 254, 254, 50);
border-radius:0;</string>
     </property>
    </widget>
    <zorder>waiter_textEdit</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>listView_2</zorder>
    <zorder>listView_3</zorder>
    <zorder>star3_pbt</zorder>
    <zorder>star2_pbt</zorder>
    <zorder>star4_pbt</zorder>
    <zorder>star1_pbt</zorder>
    <zorder>star5_pbt</zorder>
   </widget>
   <widget class="QWidget" name="tab_dishes">
    <attribute name="title">
     <string>评价菜品</string>
    </attribute>
    <widget class="QPushButton" name="back_btn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="next_btn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="star5_pbt_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>260</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="star4_pbt_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="star2_pbt_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>260</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="star1_pbt_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>260</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="star3_pbt_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>260</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="dishes_textEdit">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>270</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(244,244,244,100);
font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>145</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string> 菜名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="dish_name_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>145</y>
       <width>101</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QListView" name="listView_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(254, 254, 254, 50);
border-radius:0;</string>
     </property>
    </widget>
    <widget class="QListView" name="listView_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>255</y>
       <width>295</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(254, 254, 254, 50);
border-radius:0;</string>
     </property>
    </widget>
    <zorder>back_btn</zorder>
    <zorder>next_btn</zorder>
    <zorder>dishes_textEdit</zorder>
    <zorder>label_3</zorder>
    <zorder>dish_name_label</zorder>
    <zorder>listView_4</zorder>
    <zorder>listView_5</zorder>
    <zorder>star4_pbt_2</zorder>
    <zorder>star1_pbt_2</zorder>
    <zorder>star5_pbt_2</zorder>
    <zorder>star3_pbt_2</zorder>
    <zorder>star2_pbt_2</zorder>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>400</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(44, 178, 214);
border-radius:10%;
color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
QPushButton#pushButton:hover{  
background-color:rgb(255, 255, 214);  
}  </string>
   </property>
   <property name="text">
    <string>提交评价</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
